src/sdsc/__init__.py

```diff
@@ -965,20 +966,18 @@ def initialize():
     if sdsc_initialized:
         return True
 
-    # Prepare parser (add py: namespace)
-    ns = etree.FunctionNamespace('https://www.github.com/openSUSE/suse-doc-style-checker')
-    ns.prefix = 'py'
-    ns.update(dict(linenumber=linenumber, termcheck=termcheck,
-                   buildtermdata=buildtermdata, dupecheck=dupecheck,
-                   sentencelengthcheck=sentencelengthcheck,
-                   sentencesegmenter=sentencesegmenter,
-                   tokenizer=tokenizer, counttokens=counttokens,
-                   splitpath=splitpath))
-
-    global parser
-    parser = etree.XMLParser(ns_clean=True,
-                             remove_pis=False,
-                             dtd_validation=False)
+    # Update namespace dictionary with XSLT extension functions
+    # TODO: Move this into a separate file
```

Comment:
Yes, this is something as work in progress.

The dictionary should be completely inside `xslt.py`. However, I can do this only, if I can resolve all the global dependencies of the two remaining extension functions `termcheck` and `buildtermdata`.